doc/conf.py

```diff
@@ -86,6 +86,7 @@
 
 
 extensions = [
+    'bokeh.sphinxext.bokeh_plot',
```

Comment:
This is an attempt to fix 4 errors like `docstring of panel.models.widgets.CheckboxButtonGroup.button_type:4: ERROR: Unknown directive type "bokeh-plot".`.